src/APIInterfaces/BackendAPI/BackendAPIError.ts

```diff
@@ -33,6 +34,7 @@ export default class BackendAPIError extends Error {
         // Super needs to be called before I can call this
         if (isAxiosError(e)) {
             (this.data as any) = e.response?.data?.data;
+            this.status = e.response?.status;
```

Comment:
I'm back and forth here, I don't like copying the data

What if we made a getter call `getAxiosError` That returned AxiosError or undefined
the code would look like

```
get axiosError(): AxiosError | null {
if (isAxiosError(this.originalError)) {
return this.originalError;
}
return null;
}

usage would be:
error.axiosError?.status;
```

The main reason I like it is if people are accustomed to working with axios attributes there is other data they could grab

Reply:
Just quickly threw it in without testing but this seems like it would be the code:
```Typescript
get axiosError(): AxiosError | null {
    if (isAxiosError(this.originalError)) {
        return this.originalError;
    }
    return null;
}
```

And this would be the usage:
```
const temp = new BackendAPIError(new Error('tomtom'));
console.log(temp.axiosError?.response?.status);
```

Reply:
Sure, I'll add that in, but then, does it make sense to add a getter for Message and instead of having us support all types of Errors? Even if we swapped out Axios, I would expect a BackendAPIError to have a message and a status.

Reply:
That's a good point

I don't know if we can though since it is variable on super